From 1c767b51d9747f8a12ff10ba3e41ddcfe7a30e6e Mon Sep 17 00:00:00 2001
From: Andrei Suprun <andrei.suprun.by@gmail.com>
Date: Mon, 29 Mar 2021 01:32:26 +0300
Subject: [PATCH 1/2] edit first 2 lines

---
 FirstFile.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/FirstFile.txt b/FirstFile.txt
index 52c1cdc..880f16d 100644
--- a/FirstFile.txt
+++ b/FirstFile.txt
@@ -1,5 +1,5 @@
-A
-B
+1
+2
 C
 D
 E
-- 
2.24.1.windows.2

